### PR TITLE
Process agent and system probe should not log unhandled signals

### PR DESCRIPTION
**Pull request by sunhay**
### What does this PR do?

These are noisy logs that provide very little value - changing them to debug for now.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by shang-wang — APPROVED** [review]
Just want to make sure the function `handleSignals` in main that you deleted is not referred anywhere else right?

File: cmd/system-probe/main.go [hunk 1]
```diff
@@ -151,21 +149,6 @@ func gracefulExit() {
 	os.Exit(0)
 }
 
-func handleSignals(exit chan bool) {
```
Comment (p-lambert) [inline cmd/system-probe/main.go]: Was this dead code? Where was it being called? 🤔

File: pkg/process/util/signal_nowindows.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ func HandleSignals(exit chan bool) {
 			// Running docker.GetDockerStat() spins up / kills a new process
 			continue
 		default:
-			log.Warnf("Caught signal %s; continuing/ignoring.", sig)
+			log.Debugf("Caught signal %s; continuing/ignoring.", sig)
```
Comment (p-lambert) [inline pkg/process/util/signal_nowindows.go]: 👍

**Review by p-lambert — APPROVED** [review]

File: cmd/system-probe/main.go (hunk 1 above)
Comment (sunhay) [reply cmd/system-probe/main.go]: Yeah it was dead :)

**Pull request merged**